rclcpp/minimal_publisher/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.5)
+project(examples_rclcpp_minimal_publisher)
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(std_msgs REQUIRED)
+if (NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
+endif()
```

Comment:
I would suggest moving the C++ flags above the `find_package` (as it is done in all other packages) since the CMake config files could already try to use them.